heroic-core/src/main/java/com/spotify/heroic/http/HttpServer.java

```diff
@@ -89,7 +89,8 @@ public HttpServer(
         @Named("enableCors") final boolean enableCors,
         @Named("corsAllowOrigin") final Optional<String> corsAllowOrigin,
         final List<JettyServerConnector> connectors,
-        @Named("stopping") final Supplier<Boolean> stopping
+        @Named("stopping") final Supplier<Boolean> stopping,
+        @Named("tracing") final Optional<String> tracing
```

Comment:
These options come from dagger, and I don't see any provider to inject `tracing`. It also isn't used for anything so you can get rid of it.